dashboard/app/models/user.rb

```diff
@@ -605,7 +605,7 @@ def self.name_from_omniauth(raw_name)
 
   CLEVER_ADMIN_USER_TYPES = ['district_admin', 'school_admin'].freeze
   def self.from_omniauth(auth, params)
-    omniauth_user = where(provider: auth.provider, uid: auth.uid).first_or_create do |user|
+    omniauth_user = find_or_create_by(provider: auth.provider, uid: auth.uid) do |user|
```

Comment:
This might not be necessary for this particular feature, and could potentially be dangerous, so I might pull this out.